Pipeline/PackageBuildOutputs/PackageBuildOutputs.groovy

```diff
@@ -652,7 +697,10 @@ def parseInput(String[] cliArgs){
 	}
 	props.buildReportOrder = buildReports
 
-
+	props.generateSBOM = (opts.sbom) ? 'true' : 'false'
```

Comment:
Please use assertions to validate that the required properties for generating the SBOM are present. See the below validation block (Line 707 onwards), similar to the ones in Wazi Deploy

Reply:
Not sure here, since both the `-s` and the `-sa` parms are optional. If `-s` is provided but not `-sa`, there is a warning message issued.

Reply:
So, we don't make that a mandatory field then. I am ok with that.